cpp/src/gandiva/function_registry_math_ops.cc

```diff
@@ -97,6 +97,13 @@ std::vector<NativeFunction> GetMathOpsFunctionRegistry() {
                                        decimal128),
       BINARY_SYMMETRIC_SAFE_NULL_NEVER_FN(nvl, {}),
 
+      NativeFunction("conv", {}, DataTypeVector{utf8(), int32(), int32()}, utf8(),
+                     kResultNullIfNull, "conv_utf8_int32_int32",
+                     NativeFunction::kNeedsContext | NativeFunction::kCanReturnErrors),
+      NativeFunction("conv", {}, DataTypeVector{int64(), int32(), int32()}, utf8(),
```

Comment:
If it is easy to create a function that receives **int32,int32,int32** too, it would be good because a customer can call the function using those parameters and think that it is an error.

Reply:
Done